### ISSUE TITLE
Compile Error " XXX has no member named ‘GetFHE’"

### ISSUE THREAD

**Issue by JerryFishY** [narger-ef/LowMemoryFHEResNet20#8]
I encountered an error: 
``` C++
error: ‘using element_type = class lbcrypto::SchemeBase<lbcrypto::DCRTPolyImpl<bigintdyn::mubintvec<bigintdyn::ubint<long unsigned int> > > >’ {aka ‘class lbcrypto::SchemeBase<lbcrypto::DCRTPolyImpl<bigintdyn::mubintvec<bigintdyn::ubint<long unsigned int> > > >’} has no member named ‘GetFHE’
  386 | derivedPtr = dynamic_cast<FHECKKSRNS*>(context->GetScheme()->GetFHE().get()); 
```
while compiling the project. It seems the GetFHE method does not exist. I was wondering if it is due to the mismatch between the OpenFHE lib (v1.1.3) and the one that was used during dev since the OpenFHE is still under heavy dev.

**Comment by narger-ef** [comment]
Hello,

thank you very much for opening the issue.

Indeed, I used a custom forked version of OpenFHE in order to have access to `GetFHE()` function.
I commented those two lines in the last commit, they were supposed to clear from memory some bootstrapping pre computations ($\approx 1$GB of data), it should now work with any OpenFHE instance.

Feel free to close the issue if you are able to compile.
